frontend/src/components/CodeEditor.tsx

```diff
@@ -21,8 +21,9 @@ const CodeEditor = ({ authToken }: CodeEditorProps) => {
     const [monacoInstance, setMonacoInstance] = useState<Monaco | null>(null);
     const [output, setOutput] = useState<{ stdOut: string; stdErr: string; exitCode: number } | null>(null);
     const [error, setError] = useState<{ message: string } | null>(null);
-
+    const [currentText] = useState(typeof window !== "undefined" ? localStorage.getItem("code") || '// Write your code here' : "// Write your code here");
```

Comment:
Since your are not using `setCurrentText()`, you don't really need `useState()` to begin with.

Just pass the text you want directly into the `defaultValue` prop.